app.rb

```diff
@@ -0,0 +1,17 @@
+require 'sinatra/base'
+
+class Chitter < Sinatra::Base
+
+  get '/' do
+    "Welcome to Chitter"
+  end
+
+  get '/peep' do
+    erb :peep
+  end
+
+  post '/peeproom' do
+    Peep.all
```

Comment:
It doesn't look like you're using it right now, but in case the value was needed, this statement on its own would do nothing as the value is not being stored anywhere

```rb
@peeps = Peep.all
erb(:peeproom)
```
Would let you access e.g. `<%= @peeps %>` in the view

Reply:
> NameError at /peeproom
> uninitialized constant Chitter::Peep

Reply:
By the way, a POST route should just act on things (e.g. models/databases), not serve up views - it should redirect to a GET route. That prevents the problem where if you reload the page you send another POST request (and accidentally buy the same sofa twice).